packages/venia-concept/src/components/Checkout/addressForm.js

```diff
@@ -27,180 +26,176 @@ const fields = [
     'telephone'
 ];
 
-const filterInitialValues = memoize(values =>
-    fields.reduce((acc, key) => {
-        acc[key] = values[key];
-        return acc;
-    }, {})
-);
+const AddressForm = props => {
+    const {
+        cancel,
+        countries,
+        isAddressIncorrect,
+        incorrectAddressMessage,
+        initialValues,
+        submit,
+        submitting
+    } = props;
 
-class AddressForm extends Component {
-    static propTypes = {
-        cancel: func.isRequired,
-        classes: shape({
-            body: string,
-            button: string,
-            city: string,
-            email: string,
-            firstname: string,
-            footer: string,
-            lastname: string,
-            postcode: string,
-            region_code: string,
-            street0: string,
-            telephone: string,
-            textInput: string,
-            validation: string
-        }),
-        incorrectAddressMessage: string,
-        submit: func.isRequired,
-        submitting: bool,
-        countries: array
-    };
+    const classes = mergeClasses(defaultClasses, props.classes);
+    const validationMessage = isAddressIncorrect
+        ? incorrectAddressMessage
+        : null;
 
-    static defaultProps = {
-        initialValues: {}
-    };
+    const values = useMemo(
+        () =>
+            fields.reduce((acc, key) => {
+                acc[key] = initialValues[key];
+                return acc;
+            }, {}),
+        [initialValues]
+    );
 
-    validationBlock = () => {
-        const { isAddressIncorrect, incorrectAddressMessage } = this.props;
-        if (isAddressIncorrect) {
-            return incorrectAddressMessage;
-        } else {
-            return null;
-        }
-    };
+    const handleCancel = useCallback(() => {
+        cancel();
+    }, [cancel]);
```

Comment:
Yea I had this same feedback given on another PR. The only reason for renaming the callback would be so that it falls into some paradigm we use throughout the application. Of course, it's technically overkill and unnecessary. I am totally fine removing this sort of use of `useCallback` with a direct passing of the prop.